### DISCUSSION TITLE
Dark Mode as Default

### DISCUSSION THREAD

**Discussion by gotzsys** [filamentphp/filament#9156] [Help] [answered]
Hello everyone
I'd like to always set dark mode as the default, even if the user's system setting is white. Is it possible? However, I want to leave the option open to switch to white.

At the moment, what happens is that even when setting the default to dark, if the user's system setting is white, that option overrides the configuration.

**Comment by leandrocfe** [comment]
I'm on mobile now. I think `$panel->darkMode(true, true)` will remove the light mode option, right?

**Reply by gotzsys** [reply]
Exactly @leandrocfe , I am currently using this option, but it disables the theme selection menu.

**Reply by leandrocfe** [reply, marked as answer]
I think you could do this in the AppServiceProvider

```php
public function boot(): void
{
    FilamentView::registerRenderHook(
        'panels::scripts.after',
        fn (): string => Blade::render('
        <script>
            if(localStorage.getItem(\'theme\') === null) {
                localStorage.setItem(\'theme\', \'dark\')
            }
        </script>'),
    );
}
```

**Reply by gotzsys** [reply]
It worked perfectly, thank you very much @leandrocfe
